contracts/src/LightClient.sol

```diff
@@ -76,7 +79,9 @@ contract LightClient {
     /// @notice Wrong plonk proof or public inputs.
     error InvalidProof();
 
-    constructor(LightClientState memory genesis, uint32 numBlockPerEpoch) {
+    /// @dev Note that numBlockPerEpoch is ignore for now. See
+    /// https://github.com/EspressoSystems/espresso-sequencer/issues/940
+    constructor(LightClientState memory genesis, uint64 numBlockPerEpoch) {
```

Comment:
npoe, we shouldn't ignore, we should simply set it to max during deployment script. 

if we set it to max, then lots of our tests make no sense anymore. (cuz they are supposed to test when epoch changes, the stake table commitments gets updated etc)

Reply:
Fixed in eede4e8.

Reply:
Comment removed in 0c54374.